gulpfile.js

```diff
@@ -0,0 +1,44 @@
+var gulp                = require('gulp'),
+    livereload          = require('gulp-livereload'),
+    nodemon             = require('gulp-nodemon'),
+    shell               = require('gulp-shell'),
+    chalk               = require('chalk'),
+    paths,
+    nodemonServerInit;
+
+paths = {
+    server: 'core/server/',
+    client: 'core/client/'
+};
+
+nodemonServerInit = function () {
+    livereload.listen();
+
+    return nodemon({
+        script: 'index.js',
+        ext: 'js,json',
+        watch: [
+            'core/index.js',
+            paths.server + '**/*.js',
+            'core/built/assets/*.js'
+        ],
+        ignore: [
+            'core/client/*',
+            'core/server/test/*'
+        ]
+    }).on('restart', function () {
+        console.info(chalk.cyan('Restarting Ghost due to changes...'));
+        gulp.src('index.js')
+            .pipe(livereload());
+    });
+};
+
+// Starting Ember for development
+gulp.task('admin', shell.task(['cd ' + paths.client + ' && ember build --watch']));
+
+gulp.task('serve', function () {
```

Comment:
I think we're crossing wires here. I'm a server-side dev, I rarely ever code anything on the client, so I don't need ember to be built or watched? I just want the nodemon part that watches the server-side code for changes, and reloads.

Meanwhile, @kevinansfield is the reverse, he hardly ever needs to have nodemon sat there watching whilst he's working on the client.

I think it would be good to be able to do one or the other, instead of only both??

As this is setup, `gulp serve` does the server-side watch & reload and `gulp admin` does the admin watch & reload. My suggestion was to add one character and call `gulp serve` `gulp server`.